x/distribution/keeper/genesis.go

```diff
@@ -36,12 +36,12 @@ func (k Keeper) GetAllDelegationDistInfos(ctx sdk.Context) (ddis []types.Delegat
 // Get the set of all delegator-withdraw addresses with no limits, used during genesis dump
 func (k Keeper) GetAllDelegatorWithdrawInfos(ctx sdk.Context) (dwis []types.DelegatorWithdrawInfo) {
 	store := ctx.KVStore(k.storeKey)
-	iterator := sdk.KVStorePrefixIterator(store, DelegationDistInfoKey)
+	iterator := sdk.KVStorePrefixIterator(store, DelegatorWithdrawInfoKey)
 	defer iterator.Close()
 
 	for ; iterator.Valid(); iterator.Next() {
 		dw := types.DelegatorWithdrawInfo{
-			DelegatorAddr: sdk.AccAddress(iterator.Key()),
+			DelegatorAddr: sdk.AccAddress(iterator.Key()[1:]),
```

Comment:
This should be some common function, not a one-off.

Reply:
Moved to a common function.